### DISCUSSION TITLE
Monster Hunter World joypad not working

### DISCUSSION THREAD

**Discussion by Stooovie** [Whisky-App/Whisky#467] [General]
Monster Hunter World runs incredibly well on a M1 Pro MBP with Whiskey 2.0.0 with Steam, but anyone any idea how to make either Xbox One or PS4 controller work? They work with other games in Whiskey and Steam itself, so connection to Mac is not the issue. I have tried various layouts, Steam Input on/off, to no avail.

SOLUTION: HA! Steam Input has to be off in game Properties - Controller - Disable Steam Input. The gamepad settings being at three different places...

**Comment by Stooovie** [comment]
I see MHW is listed at Applegamingwiki as not working (black screen) - it works very well, black screen will resume in 60-90s.

**Reply by sentillier** [reply]
I've tried this solution, but it doesn't fix the problem.
The b key of the xbox one controller is mapped to the confirm key, and the left joystick cannot control the action of the character.

**Reply by jfreund01** [reply]
@sentillier ever find a solution to this?

**Comment by vcmt794** [comment]
I Can't even run MHW on Whiskey with Steam (Yes, i waited for about 5 minutes black screen and the game show up, but then crash :((( ) (Use Mac M1 Pro) :"), Can u show me your config pls?

**Reply by sentillier** [reply]
I'm using a macmini m2 Pro with the default config😂
